helm/solr-operator/README.md

```diff
@@ -191,6 +191,10 @@ The command removes all the Kubernetes components associated with the chart and
 | tolerations | []object |  | Specify a list of Kubernetes tolerations for the Solr Operator pod |
 | priorityClassName | string | `""` | Give a priorityClassName for the Solr Operator pod |
 | sidecarContainers | []object |  | An optional list of additional containers to run along side the Solr Operator in its pod |
+| logger.devel | boolean | `false` | Enable development mode for the Solr Operator logger. This will log at the debug level with the console encoder. Default is `false` which logs at `info` level with `json` encoder. |
```

Comment:
[0] I could see the dev/prod distinction being useful for setting defaults totally unrelated to logging (docker image tag, resource limits, etc.).  Wdyt about moving this "mode" flag up to the top level, so it's easier to expand later to cover other things if desired?